pkg/cmd/browse/browse.go

```diff
@@ -53,7 +55,8 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 		PathFromRepoRoot: func() string {
 			return f.GitClient.PathFromRoot(context.Background())
 		},
-		GitClient: &localGitClient{client: f.GitClient},
+		GitClient:       &localGitClient{client: f.GitClient},
+		HasRepoOverride: false,
```

Comment:
Since the default value for a boolean is `false` I think we can remove this line as it is not providing any value.